app/templates/macros/toolkit_forms.html

```diff
@@ -1,9 +1,9 @@
-{% macro text(question_content, service_data, errors, question_number=None, get_question=None) -%}
```

Comment:
What is the standard python way of doing things?  And if we're in a Jinja template does it still apply?  I only just found out how kwargs works in Jinja so may well be abusing it here through not knowing any better.  But it seemed like a neater solution than adding unused arguments to all macros.